common.sh

```diff
@@ -160,9 +175,247 @@ exit_fixed()
     exit $RESULT_FIXED
 }
 
+_get_cached_command()
+{
+    # Function gets a cached command
+    # input parameter is command provided by get_cached function
+    command=$1
+    if [ ! -f "$command" ]; then
+        log_error "File $command does not exist. It is mandatory."
+    else
```

Comment:
User did not ask for file but cached command output.  The message should reflect that.